aat/src/aat/resources/features/F-089/F-089.feature

```diff
@@ -0,0 +1,17 @@
+@F-089
+Feature: F-089: Get Case Type with Organisation and OrganisationPolicy fields
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-089.1
+  Scenario: must return case type details for the request
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains a valid jurisdiction id]
+    And the request [contains a valid caseType id]
```

Comment:
And the request [contains id of a case type with Organisation and OrganisationPolicy fields]

Reply:
The above would be good to make the scenario tell us what makes this that for the feature titles above.